plat/rockchip/common/aarch64/plat_helpers.S

```diff
@@ -35,6 +35,9 @@
 #include <cortex_a72.h>
 #include <plat_private.h>
 #include <platform_def.h>
+#if NEED_PMU_MACROS_S
+#include <plat_pmu_macros.S>
+#endif
```

Comment:
minor: Conditionally including this header isn't really necessary - it just clutters the code.